metrics/deploy/prometheus-ocs-rules.yaml

```diff
@@ -354,6 +354,16 @@ spec:
         floor((ocs_storage_provider_operator_version>0)/1000) - ignoring(storage_consumer_name) group_right() floor((ocs_storage_client_operator_version>0)/1000) > 1 or floor((ocs_storage_client_operator_version>0)/1000) - ignoring(storage_consumer_name) group_left() floor((ocs_storage_provider_operator_version>0)/1000) >= 1
       labels:
         severity: critical
+    - alert: StorageQuotaUtilizationThresholdReached
+      annotations:
+        description: Storage Client ({{ $labels.storage_consumer_name }}) has reached more than 80% of storage. Client might face issue while creating new pvc more than the quota allotted.
```

Comment:
1. We don't call the clients "Storage Clients" but "ODF Clients" 
2. The storage consumer name is not a good identifier for anything, please use the client name instead, maybe in conjunction with the cluster name (all of these should be available on the consumer's status section)
3. Please use the terminology of consumption instead of reaching. Maybe something like
``` ODF client XXXXXX storage consumption has surpassed 80% of the configured quota, this might result in issues in the creation of new PV/PVCs on the target cluster```